test/index.test.js

```diff
@@ -89,7 +89,7 @@ describe('index', function () {
     });
 
     describe('parseUrl', () => {
-        const apiUrl = 'https://gitlab.com/api/v3/projects/batman%2Ftest';
+        const apiUrl = 'https://gitlab.com/api/v4/projects/batman%2Ftest';
```

Comment:
we're not actually hitting gitlab.com are we in our tests?

Reply:
I'm fairly certain we aren't. Seems like request is mocked out for all tests.

There are a few tests still failing however, may add an extra commit to fix those.

Reply:
Thanks @Filbird.